### DISCUSSION TITLE
how to update bromite?

### DISCUSSION THREAD

**Discussion by mrawbabylon** [bromite/bromite#1145] [Q&A] [answered]
sorry if this is stupid... i don't know how to update bromite... it's an APK type of thing, right? so it won't appear in the play store and update itself for me... i just don't know how to update it - does it update itself? do i need to redownload the newest version and set it up again? i've been having a problem ([linked here](https://github.com/bromite/bromite/discussions/1143)) and i'm hoping that updating it will help me. sorry if this is a stupid question :( i don't really understand these things..

EDIT: in the linked post, i said i'd try to make a post in issues, but the reason i didn't do that is because it asked if i was on the latest one and since my bromite app says it is on bromite 78.0.3904.63, i don't think that's recent at all

**Comment by chromer030** [comment, marked as answer]
For updating bromite you have two option :  
1- Downloading latest APK, appropriate of your phone architecture (arm or arm64) from here https://github.com/bromite/bromite/releases (xx_ChromePublic.apk) and install it manually  
2- Updating through Fdroid by adding Bromite repo.  
I recommend first one , Just download APK and install it , Now you have latest version.

**Reply by mrawbabylon** [reply]
ok i'll do that, thanks for the simple answer... :D
